content/_index.md

```diff
@@ -67,8 +67,10 @@ and create [Bodhi](https://bodhi.fedoraproject.org) updates! [Learn more.](/docs
 4. _We are here to help_ - In case of questions, feel free to contact
    [us](https://github.com/orgs/packit/teams/the-packit-team):
 
-   - **#packit** on [Libera.Chat](https://libera.chat/)
-   - **hello@packit.dev**
+#### Contact
+
+- **#packit** on [Libera.Chat](https://libera.chat/)
```

Comment:
I would probably point them to the Matrix, since IRC→Matrix is not very reliable